src/screens/internal/ShareImage/ChartShareImage.style.ts

```diff
@@ -20,6 +20,15 @@ export const Title = styled.div`
   font-size: 24px;
   line-height: 18px;
   text-transform: capitalize;
+  width: 400px;
+`;
+
+export const ExploreTitle = styled(Title)`
+  font-size: 18px;
+  line-height: 19px;
+  text-transform: capitalize;
+  height: 75px;
+  overflow: hidden;
```

Comment:
I ended up reworking the styling of the title to make long / multi-line titles (with lots of locations) work better.